client/bio-commandline/src/main/kotlin/uk/ac/ebi/biostd/client/cli/services/SubmissionService.kt

```diff
@@ -40,23 +29,17 @@ internal class SubmissionService {
     private fun deleteRequest(request: DeletionRequest) =
         bioWebClient(request.server, request.user, request.password).deleteSubmissions(request.accNoList)
 
-    private fun migrateRequest(request: MigrationRequest) {
-        val sourceClient = bioWebClient(request.source, request.sourceUser, request.sourcePassword)
-        val targetClient = bioWebClient(request.target, request.targetUser, request.targetPassword)
-        val source = sourceClient.getExtByAccNo(request.accNo)
-        val sub = migratedSubmission(source, request.targetOwner)
-        val fileLists = source.allFileList
```

Comment:
Since we're not creating the file lists at this point anymore, which is great, we will no longer need the `tempFolder` parameter

https://github.com/EBIBioStudies/biostudies-backend-services/blob/master/client/bio-commandline/src/main/kotlin/uk/ac/ebi/biostd/client/cli/commands/MigrateCommand.kt#L36